Source/Controllers/SubviewControllers/SPTablesList.m

```diff
@@ -54,12 +54,13 @@
 #import "SPCharsetCollationHelper.h"
 #import "SPConstants.h"
 
+#import "Sequel_Ace-Swift.h"
+
 #import <SPMySQL/SPMySQL.h>
 
 // Constants
 //
 // Actions
-static NSString *SPAddRow         = @"SPAddRow";
```

Comment:
What happened to SPAddRow exactly?

Reply:
The key was necessary only as we were using it in separate method, but now we use callback so we don't need to handle that via keys. :)